course7/homeworks/patrickkostjens/ls/ls.cc

```diff
@@ -1,13 +1,60 @@
 #include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
 #include <iostream>
 
+namespace fs = boost::filesystem;
+namespace po = boost::program_options;
+
+size_t get_size(fs::path path)
```

Comment:
```fs::path``` is a class  that's potentially expensive to copy. You should always try to pass such objects by const reference, to save the copying